.gitignore

```diff
@@ -68,3 +68,4 @@ storybook-static
 package-lock.json
```

Comment:
should we also commit `package-lock.json` ? It seems snapshot is failing because of that

Reply:
I would stick with yarn and not include this file.

Reply:
Ah sorry I thought it uses npm